src/core/comm/coll.cc

```diff
@@ -245,19 +245,21 @@ int collInit(int argc, char* argv[])
   int provided, init_flag = 0;
   CHECK_MPI(MPI_Initialized(&init_flag));
   if (!init_flag) {
-    log_coll.fatal(
+    // TODO: check if -ll:networks none is enabled
+    log_coll.warning(
       "MPI has not been initialized, it should be initialized by "
-      "the networking backend");
+      "the networking backend, if -ll:networks none is passed, "
+      "then this warning can be safely ignored.");
```

Comment:
I would prefer to avoid emitting the warning altogether in that case. If there's a way to ask Realm whether a networking module is loaded, then if Realm answers "no" we know that the user passed `-ll:networks none`.

Reply:
Removed the warning. Now I will check if `LEGATE_NEED_NETWORK` is larger than 1, we should not init MPI ourselves, but throw an error to let people know they should not use `-ll:networks none` with more than one node.